package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -1194,12 +1212,16 @@ def _getMeanOnePoint(self,universe,selection1,selection_str,wint,totalFrames):
                 continue
             sumDeltaP += a
             n += 1
-
-        return sumDeltaP/n
+        
+        #if no water molecules remain in selection, there is nothing to get the mean, so n = 0.
+        if n == 0:
+            return (0,0,0)
```

Comment:
This is weird: isn't `sumDeltaP` a scalar? Then why would you return a tuple for `n==0`??